### PR TITLE
Convert resource-quota and rethinkdb to v1beta3

### PR DESCRIPTION
**Pull request by dchen1107**
#3671

### CODE REVIEW

File: examples/rethinkdb/v1beta3/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+RethinkDB Cluster on Kubernetes
+==============================
+
+Setting up a [rethinkdb](http://rethinkdb.com/) cluster on [kubernetes](http://kubernetes.io)
+
+**Features**
+
+ * Auto configuration cluster by querying info from k8s
```
Comment (quinton-hoole) [inline examples/rethinkdb/v1beta3/README.md]: Should read "Auto-configuration of a cluster..."

File: examples/rethinkdb/v1beta3/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+ * Simple
+
+Quick start
+-----------
+
+**Step 1**
+
+antmanler/rethinkdb will discover peer using endpoints provided by kubernetes_ro service,
```
Comment (quinton-hoole) [inline examples/rethinkdb/v1beta3/README.md]: Should read "...will discover it's peers by using endpoints provided by the kubernetes_ro service..."

File: examples/rethinkdb/v1beta3/README.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+so first create a service so the following pod can query its endpoint
+
+```shell
+kubectl create -f driver-service.yaml
+```
+
+check out:
+
+```shell
+$kubectl get se
```
Comment (quinton-hoole) [inline examples/rethinkdb/v1beta3/README.md]: Prefer using full words to improve clarity, for example:
kubectl get service

File: examples/rethinkdb/v1beta3/README.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+**Step 2**
+
+start fist server in cluster
+
+```shell
+kubectl create -f rc.yaml
+```
+
+Actually, you can start servers as many as you want at one time, just modify the `replicas` in `rc.ymal`
```
Comment (quinton-hoole) [inline examples/rethinkdb/v1beta3/README.md]: Should be "Actually, you can start as many servers as you require, all at once.  Just modify the `replicas` field in `rc.ymal` appropriately."

File: examples/rethinkdb/v1beta3/README.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+check out again:
+
+```shell
+$kubectl get po
```
Comment (quinton-hoole) [inline examples/rethinkdb/v1beta3/README.md]: As per above comment.  Prefer full words, for example "kubectl get pod"

File: examples/rethinkdb/v1beta3/README.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+**Done!**
+
+
+---
+
+Scale
+-----
+
+You can scale up you cluster using `kubectl resize`, and new pod will join to exsits cluster automatically, for example
```
Comment (quinton-hoole) [inline examples/rethinkdb/v1beta3/README.md]: Reword/typos: "... and new pods will be added to the existing cluster automatically..."

File: examples/rethinkdb/v1beta3/README.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+POD                                    IP                  CONTAINER(S)        IMAGE(S)            HOST                LABELS                       STATUS
+99f6d361-abd6-11e4-a1ea-001c426dbc28   10.240.2.68         rethinkdb           rethinkdb:1.16.0    10.245.2.2/         db=rethinkdb,role=replicas   Running
+d10182b5-abd6-11e4-a1ea-001c426dbc28   10.240.26.14        rethinkdb           rethinkdb:1.16.0    10.245.2.4/         db=rethinkdb,role=replicas   Running
+d101c1a4-abd6-11e4-a1ea-001c426dbc28   10.240.11.14        rethinkdb           rethinkdb:1.16.0    10.245.2.3/         db=rethinkdb,role=replicas   Running
+```
+
+Admin
+-----
+
+You need a separate pod (which labled as role:admin) to access Web Admin UI
```
Comment (quinton-hoole) [inline examples/rethinkdb/v1beta3/README.md]: Typo's/spelling: "You will need a separate pod (labelled as role:admin) to access the Web Admin user interface."

File: examples/rethinkdb/v1beta3/README.md [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+```shell
+kubectl create -f admin-pod.yaml
+kubectl create -f admin-service.yaml
+```
+
+find the service
+
+```shell
+$kubectl get se
```
Comment (quinton-hoole) [inline examples/rethinkdb/v1beta3/README.md]: As above. "kubectl get service"

**Comment by dchen1107** [comment]
@quinton-hoole the issues you pointed out above are existing in the original v1beta1 / v1beta2 example. I just simply copied README and yaml/json files over to v1beta3 directory. I did notice some of those problems you pointed out here, but mind to send a separate pr for those?

File: examples/rethinkdb/v1beta3/README.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+NAME                LABELS              SELECTOR                  IP                  PORT
+rethinkdb-admin     db=influxdb         db=rethinkdb,role=admin   10.241.220.209      8080
+rethinkdb-driver    db=influxdb         db=rethinkdb              10.241.105.47       28015
+```
+
+open a web browser and access to *http://10.241.220.209:8080* to manage you cluster
```
Comment (quinton-hoole) [inline examples/rethinkdb/v1beta3/README.md]: Capitalize this sentence please: "Open a web browser..."

**Comment by quinton-hoole** [comment]
@dchen1107 Sure, I can do that.  More comments once I've had a chance to test out the yaml files.

**Comment by dchen1107** [comment]
ping...

**Comment by bgrant0607** [comment]
LGTM

**Pull request merged**